File: Repl.it/Python/Гипотенуза.py
import math

print("Введите длины гипотенузы")
a = int(input())
b = int(input())
if a <= 0 | b <= 0:
    print("Нет такого треугольника")
else:
    c = math.sqrt(pow(a, 2) + pow(b, 2))
    print("Гипотенуза =  ", c)

File: Repl.it/C#/Гипотенуза.cs
using System;

class MainClass
{
    public static void Main(string[] args)
    {
        Console.WriteLine("Введите длину катетов ");
        int a, b;
        double c;
        a = Convert.ToInt32(Console.ReadLine());
        b = Convert.ToInt32(Console.ReadLine());
        if (a <= 0 || b <= 0)
            Console.WriteLine("Такого треугольника не существует");
        else
        {
            c = Math.Sqrt(a * a + b * b);
            Console.WriteLine("Длинна гипотенузы = {0}", c);
        }
    }
}